architecture/infrastructure_components/web_console.adoc

```diff
@@ -15,17 +15,16 @@ Developers can use the web console to visualize, browse, and manage the contents
 of link:../../dev_guide/projects.html[projects].
 
 The web console is started as part of the
-link:kubernetes_infrastructure.html#master[master] and by default listens one
-port higher than the API server. All static assets required to run the web
```

Comment:
Should we mention `/console` here then? Currently the only reference as to what to hit from a browser is in:

http://docs.openshift.com/enterprise/latest/dev_guide/authentication.html#web-console-authentication